### PR TITLE
Add 1.5.1 release 

### PR DESCRIPTION
**Pull request by zentol**

### CODE REVIEW

File: downloads.md [hunk 1]
```diff
@@ -104,6 +104,8 @@ Note that the community is always open to discussing bugfix releases for even ol
 
 All Flink releases are available via [https://archive.apache.org/dist/flink/](https://archive.apache.org/dist/flink/) including checksums and cryptographic signatures. At the time of writing, this includes the following versions:
 
+- Flink 1.5.1 - 2018-03-08 ([Source](https://archive.apache.org/dist/flink/flink-1.5.1/flink-1.5.1-src.tgz), [Binaries](https://archive.apache.org/dist/flink/flink-1.5.1/), [Docs]({{site.DOCS_BASE_URL}}flink-docs-release-1.5/), [Javadocs]({{site.DOCS_BASE_URL}}flink-docs-release-1.5/api/java), [ScalaDocs]({{site.DOCS_BASE_URL}}flink-docs-release-1.5/api/scala/index.html))
+- Flink 1.5.0 - 2018-03-08 ([Source](https://archive.apache.org/dist/flink/flink-1.5.0/flink-1.5.0-src.tgz), [Binaries](https://archive.apache.org/dist/flink/flink-1.5.0/), [Docs]({{site.DOCS_BASE_URL}}flink-docs-release-1.5/), [Javadocs]({{site.DOCS_BASE_URL}}flink-docs-release-1.5/api/java), [ScalaDocs]({{site.DOCS_BASE_URL}}flink-docs-release-1.5/api/scala/index.html))
```
Comment (yanghua) [inline downloads.md]: wrong date?

File: downloads.md [hunk 2]
```diff
@@ -104,6 +104,8 @@ Note that the community is always open to discussing bugfix releases for even ol
 
 All Flink releases are available via [https://archive.apache.org/dist/flink/](https://archive.apache.org/dist/flink/) including checksums and cryptographic signatures. At the time of writing, this includes the following versions:
 
+- Flink 1.5.1 - 2018-03-08 ([Source](https://archive.apache.org/dist/flink/flink-1.5.1/flink-1.5.1-src.tgz), [Binaries](https://archive.apache.org/dist/flink/flink-1.5.1/), [Docs]({{site.DOCS_BASE_URL}}flink-docs-release-1.5/), [Javadocs]({{site.DOCS_BASE_URL}}flink-docs-release-1.5/api/java), [ScalaDocs]({{site.DOCS_BASE_URL}}flink-docs-release-1.5/api/scala/index.html))
```
Comment (yanghua) [inline downloads.md]: wrong date?

File: downloads.md (hunk 1 above)
Comment (zentol) [reply downloads.md]: yes, I'll update it.

File: downloads.md (hunk 2 above)
Comment (zentol) [reply downloads.md]: Will update this once we've actually released it.

**Pull request merged**